src/components/List.js

```diff
@@ -79,6 +80,39 @@ const List = ({ token }) => {
     return daysUntilNextPurchase > 0 ? daysUntilNextPurchase : 0;
   };
 
+  const sortedList = docs.sort((a, b) => {
```

Comment:
Even though I see `sortedList` referenced, there is a message in my terminal that `sortedList is assigned a value, but never used`

<img width="642" alt="sortedListVariableNotUsed" src="https://user-images.githubusercontent.com/36529777/107452871-d2743280-6afe-11eb-82a0-ad2496bc8b5f.PNG">

Reply:
Yes, we noticed this same warning and were a little confused by it.